### PR TITLE
Fix/encoding model config

### PR DESCRIPTION
**Pull request by AlonsoGuevara**
## Description

Adding on top of #1504 
Thanks to @nreinartz

## Related Issues

[Reference any related issues or tasks that this pull request addresses.]

## Proposed Changes

[List the specific changes made in this pull request.]

## Checklist

- [x] I have tested these changes locally.
- [x] I have reviewed the code changes.
- [ ] I have updated the documentation (if necessary).
- [ ] I have added appropriate unit tests (if applicable).

## Additional Notes

[Add any additional notes or context that may be helpful for the reviewer(s).]

### CODE REVIEW

**Review by jgbradley1 — APPROVED** [review]

**Comment by jgbradley1** [comment]
Closes https://github.com/microsoft/graphrag/pull/1504

File: graphrag/config/create_graphrag_config.py [hunk 1]
```diff
@@ -626,7 +645,7 @@ def hydrate_parallelization_params(
         summarize_descriptions=summarize_descriptions_model,
         umap=umap_model,
         cluster_graph=cluster_graph_model,
-        encoding_model=encoding_model,
+        encoding_model=global_encoding_model,
```
Comment (natoverse) [inline graphrag/config/create_graphrag_config.py:648]: shouldn't this use `encoding_model`, which if unset locally has been set to the global var?

**Review by andresmor-ms — APPROVED** [review]

File: graphrag/config/create_graphrag_config.py (hunk 1 above)
Comment (natoverse) [reply graphrag/config/create_graphrag_config.py:648]: (the defaulting happens on line 473)

**Review by natoverse — CHANGES_REQUESTED** [review]

File: graphrag/config/create_graphrag_config.py (hunk 1 above)
Comment (AlonsoGuevara) [reply graphrag/config/create_graphrag_config.py:648]: Nope. It is a little confusing given how much redundancy is there with encoding config, but this one is the top level one. Line 473 is the local encoding config for claim extraction

Comment (natoverse) [reply graphrag/config/create_graphrag_config.py:648]: ok

**Review by natoverse — APPROVED** [review]

**Pull request merged**